package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "calendar-root",
+  "private": true,
+  "workspaces": [
+    "calendar-server",
+    "calendar-app"
+  ],
+  "scripts": {
+    "install:all": "npm install --workspaces",
```

```suggestion
    "install": "npm install --workspaces",
```
It is a common practice that scripts like "install", "build" or "test" at the root of the workspace runs the same scripts across all of the packages, no need to suffix them with ":all"